conf/messages

```diff
@@ -507,7 +508,7 @@ page.title.ruling.not.found=Ruling not found
 page.title.case.queues=Tariff classification case teams
 page.title.create_liability.h1=New liability information
 page.title.confirmation.h1=New case information
-page.title.create_correspondence.h1=Create correspondence case
+page.title.create_correspondence.h1=Correspondence case released to this team
```

Comment:
Hiya @oliwiaKot 
I think this title should be split as two different - one is used in the create correspondence journey, and the other one is matching the completed/release page where the h1 is this case has been created
I think best to discuss it with @shiromi  ?

Reply:
I pulled changes from https://github.com/hmrc/tariff-classification-frontend/pull/528/commits/00ef945b16bac548556437f2fd305e695656eeda like @shiro advised and also reviewed the titles on the ages I worked on and corrected them, hope this is okay now @vajarova